src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/config/RadicleSettingsView.java

```diff
@@ -189,13 +207,16 @@ public boolean isModified() {
         var selectedPath = getPathFromTextField(radPathField);
         var selectedRadHome = getPathFromTextField(radHomeField);
         return !selectedPath.equals(this.settings.getPath()) ||
-                !selectedRadHome.equals(this.settings.getRadHome());
+                !selectedRadHome.equals(this.settings.getRadHome()) ||
+                !getSelectedNodeUrl().equals(this.settings.getSeedNodes().get(0).url);
```

Comment:
`getSeedNodes()` return an empty array if the user fills something in the text field without starting with http
check `getSeedNodes()` function (`RadicleSettingsHandler.class`)